src/style.css

```diff
@@ -0,0 +1,141 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* #region reset */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+/* #endregion */
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-image {
+  display: block;
+  margin: 0 auto 40px;
+  height: 134px;
+}
+
+.card-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars__list {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+.stars__list--1 .stars__star:first-of-type,
+.stars__list--2 .stars__star:nth-of-type(-n + 2),
+.stars__list--3 .stars__star:nth-of-type(-n + 3),
+.stars__list--4 .stars__star:nth-of-type(-n + 4),
+.stars__list--5 .stars__star:nth-of-type(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__text {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: right;
+  color: #060b35;
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card-price__text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card-price__number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+.card-button {
+  display: block;
+  padding: 11px 0;
+  background-color: #00acdc; /* може не зійтись 70рх зліва */
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  text-align: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+
+  transition: all 0.5s;
```

Comment:
Owwww I didn't know you can set transition for all at once:heart_eyes::heart_eyes::heart_eyes::heart_eyes: